build_early.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+BRANCH=early
+
+cd `dirname $0`/..
+. site/build_vars.sh
+
+if [ "`git branch --show-current`" != "main" ]; then
+    echo "Not on main branch."
+    exit
+elif [ "`git status -s`" != "" ]; then
+    echo "Changes in main branch."
+    exit
+fi
+YEAR=`date +%Y`
+LASTVER=`git -C site tag -l | grep $YEAR | cut -d. -f2 | sort -g | tail -n1`
+if [ "$LASTVER" == "" ]; then
+    GLUON_RELEASE="${GLUON_RELEASE:-${YEAR}.1}-${BRANCH}"
+else
+    GLUON_RELEASE="${GLUON_RELEASE:-${YEAR}.$(( $LASTVER + 1 ))}-${BRANCH}"
+fi
+echo "Building Gluon Release ${GLUON_RELEASE} (${BRANCH}) for targets: ${TARGETS}..."
+if [ ! -f "$1" ]; then
+    echo "Manifest will not be signed, because no secret was given."
+fi
+
+echo "Cleaning Output..."
+rm -rf output
```

Comment:
I suggest to check if the directory exists:
`test -d ./output && rm -rf ./output`